src/web/views/locales/en/unsuccessful/error.html

```diff
@@ -0,0 +1,3 @@
+<p class="govuk-body">An error occurred whilst checking your eligibility</p>
+<p class="govuk-body"> <a href="apply">Go back and try again</a>.</p>
```

Comment:
Should the links have the govuk-link class attached? Or, should we finally update the css so we don't have to keep doing this.